guides/common/modules/proc_adding-errata-to-a-host.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Adding_errata_to_a_single_host_{context}"]
+= Adding errata to a single host
+
+Use the following procedure to add errata to a host.
+
+.Procedure
+. In the Satellite web UI, navigate to *Hosts > All Hosts* and select the host you want to add Errata to.
```

Comment:
```suggestion
. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and select the host you want to add Errata to.
```

Reply:
Please do not submit PRs without running checks locally for different scenarios. Every time we have to correct variables missing from your PRs. It's a waste of our time.